check_ssl.sh

```diff
@@ -3,9 +3,9 @@
 # HELP |
 #-------
 function HELP {
-NORM=`tput sgr0`
-BOLD=`tput bold`
-REV=`tput smso`
+NORM=$(tput sgr0)
+BOLD=$(tput bold)
+REV=$(tput smso)
 echo -e \\n"Help documentation for ${BOLD}$0${NORM}"\\n
```

Comment:
As we're already in the business of refactoring, I would suggest replacing all these `echo` calls with a single heredoc into `cat`, i.e.

```bash
function HELP {
	cat <<- EOF

	Help documentation for $BOLD$0$NORM
	$REV-H$NORM  -- Sets the value for...
	…
	EOF
}
```

Reply:
If we go that far with the refactor I would proceed with making it POSIX compliant.

Please don't make me do that. :sob:

Reply:
> At this point, just rewrite it in go.

Go away!

Reply:
@ayekat I refactored it in 62df36bacf6735f8ad8f5b792f5c4bccff98d2d6, but somehow `cat <<-` didn't work and I had to use it without indentation.

Reply:
The `<<-` variant only works if indented with a tab, not with spaces.

Given that the rest of the script uses tabs for indentation, I would propose (both for consistency's sake and for making the `<<-` notation work) to use tabs here as well.